lib/create-space-api.js

```diff
@@ -206,7 +206,8 @@ import omit from 'lodash/omit'
  * @return {ContentfulSpaceAPI}
  */
 export default function createSpaceApi ({
-  http
+  http,
+  httpUploads
```

Comment:
Can't you just swap configs whenever you need them  https://github.com/mzabriskie/axios#config-defaults  

Maybe a util function like `swapConfig` and when creating the client we save our two configs for upload and for normal API request. this way we won't have another axios instance, and maybe in the future the direct file Upload will be rate-limited and it is gonna be a bit hard to control the rate of requests per period of time for two axios instances